.pre-commit-config.yaml

```
@@ -48,5 +48,12 @@ repos:
       - id: golangci-lint
         args: [--new-from-patch=/tmp/diff.patch]
 
+    # Rules are in .markdownlint.json file
```

```suggestion
    # Rules are in .markdownlint-cli2.yaml file
```
Sorry just noticed this

---

Thanks for catching!